lms/product/canvas/_plugin/misc.py

```diff
@@ -3,16 +3,11 @@
 from typing import Optional
 from urllib.parse import unquote
 
-from lms.services.assignment import AssignmentService
+from lms.product.plugin.misc import MiscPlugin
```

Comment:
The diff shows this file as renamed to be the plugin which is fair enough as most of the method tried here are Canvas only.